src/tests/ftest/util/test_utils.py

```diff
@@ -345,24 +345,26 @@ def check_pool_daos_space(self, s_total=None, s_free=None):
             for index, item in enumerate(val)]
         return self._check_info(checks)
 
-    def check_rebuild_status(self, rs_version=None, rs_pad_32=None,
+    def check_rebuild_status(self, rs_version=None, rs_seconds=None,
```

Comment:
Is there any reason to not include the `rs_padding32` item?

Reply:
just missing that.